### PR TITLE
Setup Design Style

### PR DESCRIPTION
**Pull request by HowardYaw**
- Theme Color
- Font Style
- Heading Style
### Font Style Sample 
image

### CODE REVIEW

**Review by kachun333 — DISMISSED** [review]

**Comment by HowardYaw** [comment]
Font Style is updated in ea6f2d435b12eae3c668fafe5f25666e540273d0 

As suggest by @junzai97 to use font-face but the Roboto Font style is not apply when using font-face custom family name

So I just set the font weight in each heading type

File: src/main/webapp/app/app.scss [hunk 1]
```diff
@@ -1,8 +1,65 @@
+// Custom styles and variables
+@import './bootstrap-variables';
 // Override Boostrap variables
 @import '~bootswatch/dist/minty/variables';
 // Import Bootstrap source files from node_modules
 @import 'node_modules/bootstrap/scss/bootstrap';
 @import '~bootswatch/dist/minty/bootswatch';
+// Roboto Font
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
```
Comment (junzai97) [inline src/main/webapp/app/app.scss]: This links have limited font weight. Some of the font weight styling set in below does not take effect.
Solution: Change the link as following to retrieve the font with variant weight:

https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap

**Review by kachun333 — DISMISSED** [review]

File: src/main/webapp/app/app.scss (hunk 1 above)
Comment (HowardYaw) [reply src/main/webapp/app/app.scss]: resolved in 6f13656c8e062380df43ebbd395285a488f8dad4

**Review by junzai97 — APPROVED** [review]
Looks good to me!

**Review by LUXIANZE — APPROVED** [review]

**Pull request merged**